dwave/cloud/client/base.py

```diff
@@ -1088,6 +1090,11 @@ def predicate(solver, query, val):
         if 'name__eq' in filters:
             query['name'] = filters['name__eq']
 
+        # shortcircuit lookup if identity defines a name
+        identity = filters.get('identity__eq', filters.get('identity', {}))
+        if 'name' in identity:
+            query['name'] = identity['name']
```

Comment:
Seems like this could update the above code to something like:
```python
if 'name' in identity:
    query['name'] = identity['name']
elif 'name__eq' in filters:
    query['name'] = filters['name__eq']
elif 'name' in filters::
    query['name'] = filters['name']
```

Reply:
sure, I just find it more readable when separated -- one block is checking filters, the other identity